src/lib/apis/fetch.ts

```diff
@@ -80,7 +80,7 @@ export default (url, options = {}) => {
       ) {
         const message = compact([
           get(response, "request.uri.href"),
-          response.body,
+          response.statusMessage,
```

Comment:
Since `request.body` was generally already JSON-parsed, this wound up passing a string `[Object object]` around). Because the whole `response` is being passed to `HTTPError`, I don't think we need to inject a meaningful 'message' in the internally-thrown error here.